.github/workflows/on-pull-request.yml

```diff
@@ -31,6 +31,20 @@ jobs:
                   project-root: "."
                   output-dir: "."
 
+            - name: Build instructor version with PreTeXt
+              uses: siefkenj/pretext-build-action@main
+              with:
+                  pretext-command: build instructor
+                  project-root: "."
+                  output-dir: "."
+                  
+            - name: Build instructor-print version with PreTeXt

```

Comment:
No strong opinion, but do we need a print version for instructors? Or at least do we need it building on every PR?

Reply:
It was easy enough to add, and it feels like the thing that a few instructors might like.  I'm agnostic on whether they build all the time, other than making sure we don't forget to build them when deploying

Reply:
we'll just keep it in the CI for now then